packages/mongo/collection/methods_async.js

```
@@ -101,6 +129,11 @@ export const AsyncMethods = {
    * @param {Object} doc The document to insert. May not yet have an _id attribute, in which case Meteor will generate one for you.
    */
   insertAsync(doc, options) {
+    const hooks = this._hooks?.insert;
+    if (!CollectionHooks._directEnv.get() && hooks &&
```

ESLint

[error] Replace `!CollectionHooks._directEnv.get()·&&` with `⏎······!CollectionHooks._directEnv.get()·&&⏎·····`

(prettier/prettier)